release notes/v0.47.0.md

```diff
@@ -0,0 +1,92 @@
+k6 `v0.47.0` is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
```

Comment:
This time the list is already compact so probably we can omit them as it will be probably the same as the feature list. WDYT? If you want to keep then make a suggestion, please.